unfoldingWord/translationCore
# Populate wordBank with targetLanguage

#### This pull request addresses:

This uses the targetLanguage to populate the wordBank in the word alignment tool.


#### How to test this pull request:

Open a compatible project in the wordAlignment tool (after chapter index PR is merged, this can be any titus project). 

Ensure that the word bank is showing the target language words instead of ULB. 

The users will be opening the ULB as their targetLanguage to align.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2816)
